liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -1094,4 +1094,31 @@ private void runTestCompileMojoLogWarning() throws MojoExecutionException {
         runCompileMojo("testCompile");
     }
 
+    /**
+     * Executes liberty:install-feature unless using Liberty in a container
+     * @throws MojoExecutionException
+     */
+    protected void runLibertyMojoInstallFeature(Element features) throws MojoExecutionException {
+        if (!container) {
+            super.runLibertyMojoInstallFeature(features);
+        }
+    }
+
+    /**
+     * Executes liberty:create unless using a container, then just create the necessary server directories
+     * @throws MojoExecutionException
+     */
+    protected void runLibertyMojoCreate() throws MojoExecutionException {
```

Comment:
Add the `@Override` annotation to both of these methods so that a compile error would occur if something changes where it no longer overrides the parent.

Reply:
Indeed